lintro/ai/review/context/git_ops.py

```
@@ -0,0 +1,255 @@
+        Completed process with stdout/stderr captured as text.
+
+    Raises:
+        ReviewContextError: When git execution fails and ``check`` is True.
+    """
+    git_bin = _resolve_executable(
+        command="git",
+        code=ReviewContextErrorCode.GIT_UNAVAILABLE,
+        message="git is not installed or not on PATH — required for lintro review.",
+    )
+    try:
+        result = subprocess.run(  # nosec B603 - argv is [resolved git binary, *args]; shell=False; args are git subcommands only, not user shell input
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)